contracts/Overtime/SportsAMMV2.sol

```diff
@@ -100,11 +101,16 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
         uint _expectedPayout;
         uint _additionalSlippage;
         address _differentRecipient;
-        bool _sendDefaultCollateral;
         bool _isLive;
         address _requester;
+        address _collateral;
+        address _collateralPool;
+        uint _collateralPriceInUSD;
     }
 
+    mapping(address => address) public collateralPool;
+    uint public defaultCollateralDecimals;
```

Comment:
why do we need this as a state variable if it can be fetched from the manager?

Reply:
we add an external call (more gas) per trade
whereas we can set this one time and be available directly

Reply:
removed it at the end